plugins/catalog/src/components/CatalogPage/DefaultCatalogPage.test.tsx

```diff
@@ -263,17 +263,25 @@ describe('DefaultCatalogPage', () => {
     const { getByTestId } = await renderWrapped(<DefaultCatalogPage />);
     fireEvent.click(getByTestId('user-picker-owned'));
     await expect(screen.findByText(/Owned \(1\)/)).resolves.toBeInTheDocument();
-    fireEvent.click(screen.getByTestId('user-picker-starred'));
-    await expect(
-      screen.findByText(/Starred \(0\)/),
-    ).resolves.toBeInTheDocument();
+    // The "Starred" menu option should initially be disabled, since there
```

Comment:
The only change in plugin-catalog is this test change - not sure whether this requires a changeset or not.